tests/unit/operators/test_custom_models.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2023 DataRobot, Inc. and its affiliates.
```

Comment:
would be nice to add negative test cases to check behavior when user missing some required input parameters

Reply:
Added negative test cases